charts/kubeclarity/templates/scanner-template-configmap.yaml

```diff
@@ -71,6 +71,20 @@ data:
               value: {{ include "kubeclarity.grype-server.name" . }}.{{ .Release.Namespace }}:{{ index .Values "kubeclarity-grype-server" "servicePort" }}
             - name: SCANNER_REMOTE_GRYPE_SERVER_TIMEOUT
               value: {{ index .Values "kubeclarity-runtime-scan" "vulnerability-scanner" "scanner" "grype" "remote-grype" "timeout" }}
+{{- end }}
+            - name: ANALYZER_TRIVY_CACHE_DIRECTORY
```

Comment:
shouldn't `ANALYZER_TRIVY_CACHE_DIRECTORY` be configured only if trivy analyzer is enabled?